src/main/java/org/pac4j/jax/rs/annotations/Pac4JSecurity.java

```diff
@@ -0,0 +1,42 @@
+package org.pac4j.jax.rs.annotations;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.pac4j.jax.rs.filter.SecurityFilter;
+
+/**
+ * 
+ * Identify the class or method as being filtered by {@link SecurityFilter}.
+ * 
+ * @author Victor Noel - Linagora
+ *
+ */
+@Target({ ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Pac4JSecurity {
+
+    /**
+     * @return value for {@link SecurityFilter#setAuthorizers(String)}
+     */
+    String[] value() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setClients(String)}
+     */
+    String[] clients() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setMatchers(String)}
+     */
+    String[] matchers() default {};
+
+    /**
+     * @return value for {@link SecurityFilter#setMultiProfile(boolean)}
+     */
+    boolean multiProfile() default false;
```

Comment:
If for one reason or another, we decide in version 2.0 that we should be multi-profiles by default, you should change this value while it could be only changed in logics if it was `null` which is indeed not possible.

Not a big deal, just for information.